lib/ci-helper.ts

```diff
@@ -160,7 +166,14 @@ export class CIHelper {
             }
             const meta = await this.getMailMetadata(messageID);
             if (!meta || meta.commitInGitGit !== undefined) {
-                continue;
+                if (!meta || commitsInPu.has(meta.commitInGitGit!)) {
+                    continue;
+                }
+                console.log(`Upstream commit ${meta.commitInGitGit} for ${
+                    info.headCommit} of ${
+                    info.pullRequestURL} no found longer in pu`);
```

Comment:
Should probably be `no longer found in pu`.

Reply:
Oy. 🤦‍♂